docs/extensibility/extending-blocks.md

```diff
@@ -155,20 +155,20 @@ wp.hooks.addFilter( 'editor.BlockEdit', 'my-plugin/with-inspector-controls', wit
 {% ESNext %}
 ```js
 const { createHigherOrderComponent } = wp.compose;
-const { Fragment } = wp.Element;
+const { Fragment } = wp.element;
 const { InspectorControls } = wp.editor;
 const { PanelBody } = wp.components;
 
 const withInspectorControls =  createHigherOrderComponent(BlockEdit => {
```

Comment:
According to our conventions, we should use parenthesis around the function parameters e.g: ( BlockEdit ) =>, ( props ) =>.